citesphere/src/main/webapp/WEB-INF/views/auth/authorities/create.html

```diff
@@ -31,13 +36,20 @@
     });
 
     $("#searchAuthor").click(function() {
+    	$("#conceptpowerAuthoritySearchResult").empty();
         $("#selectAuthorityModel").on('hidden.bs.modal', function(e) {
             $('#selectAuthorityModel a:first').tab('show');
         })
-        $("#searchAuthorSpinner").show();
+        
+        $('#conceptpowerAuthoritiesTabContent').toggleClass('greyout');
+        $("#authoritySpinner").show();
         $('#conceptpowerAuthority-pagination-top').twbsPagination('destroy');
-        getConceptpowerAuthorities('Author', 'Author', 0)
-        $("#searchAuthorSpinner").hide();
+        
+        setTimeout(function() {getConceptpowerAuthorities('Author', 'Author', 0);}, 2000);
```

Comment:
why is this method called after 2sec?

Reply:
Just so that it can fetch all the related concept powers within 2 secs and then display them. Until then the spinner keeps running.